appchain/src/tests.rs

```diff
@@ -617,12 +618,12 @@ fn test_submit_validator_sets_on_chain() {
 
 	SyncCryptoStore::sr25519_generate_new(
 		&keystore,
-		crate::crypto::Public::ID,
+		TEST_KEY_TYPE ,
```

Comment:
As you use this in mocks:
```
use pallet_octopus_appchain::sr25519::AuthorityId as OctopusId; 
...

type RuntimeAppPublic = OctopusId ;
```
you should keep the KEY_TYPE same as in pallet_octopus_appchain::sr25519.

so, I suggest you directly use KEY_TYPE instead of defining a new one.